templates/components/itilobject/timeline/timeline_item_header_badges.html.twig

```diff
@@ -31,14 +31,11 @@
  # ---------------------------------------------------------------------
  #}
 
-<span class="badge text-wrap d-none d-md-block">
+<span class="badge user-select text-wrap d-none d-md-block">
    {% set date_span %}
       <span title="{{ date_creation|formatted_datetime }}"
-            data-bs-toggle="tooltip" data-bs-placement="bottom">
-         <i class="far fa-clock me-1"></i>
-         <a href="#{{ anchor }}">
-            {{ date_creation|relative_datetime }}
```

Comment:
This was wanted (relative dates).
If some wants precise dates, you must add a config/preference options for that

Reply:
done